google-cloud-storage/src/main/java/com/google/cloud/storage/StorageOptions.java

```diff
@@ -105,6 +118,9 @@ private StorageOptions(Builder builder, StorageDefaults serviceDefaults) {
         new RetryAlgorithmManager(
             MoreObjects.firstNonNull(
                 builder.storageRetryStrategy, serviceDefaults.getStorageRetryStrategy()));
+    this.invocationIDEnabled =
+        MoreObjects.firstNonNull(
+            builder.storageInvocationIDEnabled, serviceDefaults.getStorageInvocationIDEnabled());
```

Comment:
We won't need this after moving to `boolean`. We'll be able to update the builder to read `serviceDefaults.getStorageInvocationIDEnabled()` when constructed, and not worry about it after that point.